content/management/configuration-api/v3.2/index.mdx

```diff
@@ -2785,6 +2785,230 @@ curl -X POST \
 
 </Section>
 
+# Auto access rules
+
+Auto access rules allow to assign groups to customer session according to their tracking url and geolocation.  Those rules are
+checked on **Login** and **Update Customer Page** and resulting groups stored in customer session.  When that session starts chat with **Start Chat**
+or **Activate Chat** without `chat.access` stored session groups are used as new thread's access.
+
+## Methods
+
+<Section>
+
+<Text>
+
+### Add auto access rules
+
+
+#### Specifics
+
+|                |                                                                               |
+| -------------- | ----------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.2/configuration/action/add_auto_access_rules` |
+| Required scopes| `access_rules:rw`                                                             |
+
+
+| Parameter       | Required     | Data type | Notes                         |
+| --------------- | ------------ | --------- | ----------------------------- |
+| `access`        | Yes          | `object`  | Destination access            |
+| `access.groups` | Yes          | `[]int`   | Destination groups            |
+| `rules`__**__   | Yes          | `object`  | Rules to check                |
+| `rules.customer_url` | No      | `object`  | Rule that matches on `customer_page.url` from **Login** or `url` from **Update Customer Page** |
+| `rules.customer_domain` | No   | `object`  | Rule that matches on domain of the url above |
+| `rules.customer_*.values` | No | `[]match_object`_*_ | Positive match, may not be used together with `exclude_values` |
+| `rules.customer_*.exclude_values` | No | `[]match_object` | Negative match |
+| `rules.customer_domain` | No   | `object`  | Rule that matches on customer geolocation |
+| `rules.customer_domain.values` | Yes | `[]geolocation_object`__**__ | array of objects with optional values, at least one must be set; matches when at least one subrule matches |
```

Comment:
optionality of `geolocation` fields should be described below where the structure is presented